.github/workflows/rust-build-and-test.yml

```diff
@@ -97,7 +97,13 @@ jobs:
         else
           BUILD_TAG=$(echo ${{ github.sha }} | head -c 12)
         fi
-        make SHA_TAG="$BUILD_TAG" publish-docker
+        # Tag with latest if merge to main
+        if [ "${{ github.event_name }}" == "merge_group" ]; then
+          TAG_LATEST=true
+        else
+          TAG_LATEST=false
+        fi
+        make SHA_TAG="$BUILD_TAG" TAG_LATEST="$TAG_LATEST" publish-docker
```

Comment:
Can revert this section, actually. The `cd-to-infra.yml` workflow detects pushes to `main` and so there you could add `TAG_LATEST="true"` in the `make` command there:
```
make SHA="${{ github.SHA }}" SHA_TAG="$SHA_TAG" RELEASE_TAG="$RELEASE_TAG" TAG_LATEST="true" publish-docker
```

Reply:
Removed!

Reply:
Wait haha, I meant can keep the TAG_LATEST change in the publish script, revert changes to the test workflow yml, and then add the TAG_LATEST=true to the CD workflow.

Reply:
Ah, updated publish to set the latest tag when a release is done.

We don't need the TAG_LATEST anymore, yeah?

Updated for that.